lib/classes/account_class.rb

```diff
@@ -0,0 +1,36 @@
+module BankAccount
+
+  class Account
+    # changed to a reader since it doesn't seem like these attributes should be able to be overwritten
```

Comment:
:+1: removing the ability of external code modifying instance variables generally leads to better encapsulation and generally better object oriented design.